refac: playground renamed to domain-test (#265)
Energinet-DataHub/geh-core: source/TestCommon/documents/release-notes/release-notes.md

# TestCommon Release notes

## Version 4.4.2

> - Bump version as part of renaming Databricks keys in keyvault

Reviewer: ```suggestion
- Databricks integration test configuration keys in keyvault are renamed
```

> - Bump version as part of renaming Databricks keys in keyvault 

Reviewer: ```suggestion
- Bump version as part of renaming Databricks keys in keyvault
```
